src/main/java/com/zipcodewilmington/phonebook/PhoneBook.java

```diff
@@ -1,7 +1,60 @@
 package com.zipcodewilmington.phonebook;
 
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.TreeMap;
+
 /**
  * Created by leon on 1/23/18.
  */
 public class PhoneBook {
-}
+    private TreeMap<String, ArrayList> phoneBook;
+
+    public PhoneBook(){
+        this.phoneBook = new TreeMap<String, ArrayList>();
+    }
+
+
+
+    public void add(String name, String... phoneNumber){
+        ArrayList<String> phoneNumberList = new ArrayList<String>();
+        for (String number : phoneNumber){
+            phoneNumberList.add(number);
+        }
+        phoneBook.put(name, phoneNumberList);
+    }
+
+    public String lookUp(String name){
+        String result = String.valueOf(phoneBook.get(name));
+        return result.substring(1, result.length()-1);
+    }
```

Comment:
I guess Iwould return [0] not len()-1, but...

Reply:
It prints out with brackets so it looks like "[2154637564]" and i was trying to remove the brackets

Reply:
yeah, but handling a clean formatting would be more to the spirit of the usage of the class.